src/storybook/html.stories.tsx

```diff
@@ -7,6 +7,41 @@ const storyMeta: Meta = {
   title: 'Demo/Html',
 }
 
+const Content: React.FC<{ text: string }> = ({ text }) => {
+  const [count, setCount] = React.useState(1)
+
+  React.useEffect(() => {
+    const id = setInterval(() => {
+      setCount((cur) => (cur > 5 ? 1 : cur + 1))
+    }, 1000)
+
+    return () => clearInterval(id)
+  }, [])
+
+  const cats = ['😹', '😻', '🙀', '😸', '😺', '😼']
+
+  return (
+    <div
+      style={{
+        color: 'white',
+        display: 'flex',
+        placeContent: 'center',
+        flexDirection: 'column',
+        height: '100%',
+        textAlign: 'center',
+      }}
+    >
+      <h1 style={{ fontWeight: 'normal' }}>
+        {Array.from(
+          { length: count },
+          () => cats[Math.floor(Math.random() * cats.length)],
+        )}
+      </h1>
+      {text}
+    </div>
+  )
+}
```

Comment:
https://dromru.github.io/react-photoswipe-gallery/slide-content/?path=/story/demo-html--html

something went wrong with story source code addon

<img width="531" alt="source code is not valid" src="https://user-images.githubusercontent.com/37641303/164122315-71456593-022b-4216-9630-a11028408a47.png">

Reply:
It's fine 😆